admin_guide/overcommit.adoc

```diff
@@ -45,68 +45,39 @@ lowest quality of service.
 [[tune-buffer-chunk-limit]]
 === Tune Buffer Chunk Limit
 
-If Fluentd logger is unable to keep up with a high number of logs, you will need
-to increase the compute resource values.
+If Fluentd logger is unable to keep up with a high number of logs, it will need
+to switch to file buffering to reduce memory usage and prevent data loss.
 
-The memory limit is used to calculate the Fluentd `buffer_queue_limit` as follows:
+The Fluentd `buffer_chunk_limit` is determined by the environment variable
+`BUFFER_SIZE_LIMIT`, which has the default value 8m. The file buffer size per
+output is determined by the environment variable `FILE_BUFFER_LIMIT`, which has
+the default value 256Mi. The permanent volume size has to be larger than
+`FILE_BUFFER_LIMIT` times number of output.
 
-----
-buffer_queue_limit = resource memory limit / (number of output * buffer_chunk_size)
-----
-
-By default, `buffer_chunk_size` is 1 MB.
-
-The following steps allow you to adjust the available resources.
-
-. Edit the daemonset of Fluentd:
-+
-----
-$ oc edit daemonset logging-fluentd
-
-resources:
-  limits:
-    cpu: 100m
-    memory: 512Mi
-----
-
-. Increase the values according to available resources. For example:
-+
-----
-resources:
-  limits:
-    cpu: 150m
-    memory: 1Gi
-----
+On the Fluentd and Mux pods, permanent volume `/var/lib/fluentd` should be
+prepared by the PVC or hostmount, for example. That area is then used for the
+file buffers.
 
-If the `mux` server is behind the incoming logs, the same configuration is
-avaialable. The memory limit is used to calculate the `mux` `buffer_queue_limit`
-as follows:
+The `buffer_type` and `buffer_path` are configured in the Fluentd config files as
+follows:
 
 ----
-buffer_queue_limit = resource memory limit / (number of output * buffer_chunk_size)
-----
-
-By default, `buffer_chunk_size` is 1 MB.
-
-. Edit the deploymentconfig of `mux`:
-+
+$ egrep "buffer_type|buffer_path" *.conf
+output-es-config.conf:
+  buffer_type file
+  buffer_path `/var/lib/fluentd/buffer-output-es-config`
+output-es-ops-config.conf:
+  buffer_type file
+  buffer_path `/var/lib/fluentd/buffer-output-es-ops-config`
+es-copy-config.conf:
```

Comment:
The example is LGTM,  but es-copy-config.conf and es-ops-copy-config.conf is deprecated
@nhosoi 
What about delete these two config files from this doc, and add
filter-pre-mux-client.conf:
  buffer_type file
  buffer_path `/var/lib/fluentd/buffer-mux-client`

Reply:
@juzhao 
+1 on your proposals.

As of release-3.7, we stopped including es-copy-config.conf and es-ops-copy-config.conf.
Regarding filter-pre-mux-client.conf, it was introduced since release-3.6.

Reply:
Thanks, @juzhao and @nhosoi! PR updated